crowbar_framework/app/models/backup.rb

```diff
@@ -32,16 +33,11 @@ class Backup < ActiveRecord::Base
       message: "allows only letters and numbers"
     }
 
-  validates :version,
-    presence: true
-
-  validates :size,
-    presence: true
```

Comment:
I don't understand why this gets removed. Can this  be explained in the commit message?